docs/developer/contributing.rst

```diff
@@ -0,0 +1,185 @@
+
+If you have authored a commit that is missing the signed-off-by line, you can amend your
+commits and push them to GitHub.
+
+.. code-block:: bash
+
+   git commit --amend --signoff
+
+If you've pushed your changes to GitHub already you'll need to force push your branch
+after this with ``git push -f``.
+
+Commit Message formatting
+-------------------------
+
+To ensure that all commit messages in the master branch follow a specific format, we
+enforce that all commit messages must follow the following format:
+
+.. code-block:: bash
+
+    FEAT-#9999: Add some functionality to enable something
+
+The ``FEAT`` component represents the type of commit. This component of the commit
+message can be one of the following:
+
+* FEAT: A new feature that is added
+* DOCS: Documentation improvements or updates
+* FIX: A bugfix contribution
+* REFACTOR: Moving or removing code without change in functionality
+* TEST: Test updates or improvements
+
+The ``#9999`` component of the commit message should be the issue number in the unidist
+GitHub issue tracker: https://github.com/modin-project/unidist/issues. This is important
+because it links commits to their issues.
+
+The commit message should follow a colon (:) and be descriptive and succinct.
+
+General Rules for Committers
+----------------------------
+
+- Try to write a PR name as descriptive as possible.
+- Try to keep PRs as small as possible. One PR should be making one semantically atomic change.
+- Don't merge your own PRs even if you are technically able to do it.
+
+Development Dependencies
+------------------------
+
+We recommend doing development in a virtualenv or conda environment, though this decision
+is ultimately yours. You will want to run the following in order to install all of the required
+dependencies for running the tests and formatting the code:
+
+.. code-block:: bash
+
+  conda env create --file environment.yml
+  # or
+  pip install -r requirements.txt
+
+Code Formatting and Lint
+------------------------
+
+We use black_ for code formatting. Before you submit a pull request, please make sure
+that you run the following from the project root:
+
+.. code-block:: bash
+
+  black .
+
+We also use flake8_ to check linting errors. Running the following from the project root
+will ensure that it passes the lint checks on Github Actions:
+
+.. code-block:: bash
+
+  flake8 .
+
+We test that this has been run on our `Github Actions`_ test suite. If you do this and find
+that the tests are still failing, try updating your version of black and flake8.
+
+Adding a test
+-------------
+
+If you find yourself fixing a bug or adding a new feature, don't forget to add a test to
+the test suite to verify its correctness! We ask that you follow the existing
+structure of the tests for ease of maintenance.
+
+Running the tests
+-----------------
+
+To run the entire test suite, run the following from the project root:
+
+.. code-block:: bash
+
+  pytest unidist/test
```

Comment:
```suggestion
  python unidist/cli -m pytest unidist/test
```
or
```suggestion
  unidist -m pytest unidist/test
```

We should describe a general way of running any unidist program to make it possible to run tests on any backend.

Reply:
Fixed. Added a line about CLI and a comment on inserting a link to the CLI.